src/models/iconSchema/iconDefinition.ts

```diff
@@ -0,0 +1,15 @@
+import { IIconPath } from './iconPath';
+
+export interface IIconDefinition {
+  _file: IIconPath;
+  _folder: IIconPath;
+  _folder_open: IIconPath;
+  _file_light: IIconPath;
+  _folder_light: IIconPath;
+  _folder_light_open: IIconPath;
+  _root_folder: IIconPath;
+  _root_folder_open: IIconPath;
+  _root_folder_light: IIconPath;
+  _root_folder_light_open: IIconPath;
+
```

Comment:
is that whitespace by design?

Reply:
No. Will fix it tomorrow along with the test failure.

Reply:
Fixed.